src/components/study/detail/StudyDetailPageApplicant.tsx

```
@@ -34,6 +35,9 @@ const StudyDetailPageApplicant: React.FC<DetailPageProps> = ({ studyDetail }) =>
       console.error('Error creating comment:', error);
     }
   };
+  */}
+
+  const handleSubmit = () => {};
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)